docs/concepts/work-pools.md

```diff
@@ -262,3 +269,95 @@ Together work queue priority and concurrency enable precise control over work. F
 
 ### Local debugging
 As long as your deployment's infrastructure block supports it, you can use work pools to temporarily send runs to an agent running on your local machine for debugging by running `prefect agent start -p my-local-machine` and updating the deployment's work pool to `my-local-machine`.
+
+## Workers Overview <span class="badge beta"></span>
+
+!!! warning "Workers are a beta feature"
+    Workers are a beta feature and are subject to change in future releases.
+
+Workers are lightweight polling services that retrieve scheduled work from a work pool and execute the corresponding flow runs. Workers are similar to agents but offer greater configurability and the ability to route work to specific execution environments.
+
+Workers each have a type corresponding to the execution environment to which they will submit flow runs. Workers are only able to join work pools of the same type. As a result, when deployments are assigned to a work pool, you know in which execution environment scheduled flow runs for that deployment will be scheduled.
+
+### Worker Options
+
+Workers are configured to pull work from one or more queues within a work pool. If the worker references a work queue that doesn't exist, it will be created automatically. If a worker references a work pool that doesn't exist, it will automatically create it as long as the type is provided. 
+
+Configuration parameters you can specify when starting a worker include:
+
+| Option | Description |
+| --- | --- |
+| `--name`, `-n` | The name to give to the started worker. If not provided, a unique name will be generated. |
+| `--pool`, `-p` | The work pool the started worker should join. |               
```

Comment:
@desertaxle Its not clear to me whether "join" or "poll" fits best. "Join" feels more like there is existing work that could be spread out over the workers. "poll" evokes a queuing concept. I think I prefer "poll" but I could be convinced otherwise.

Reply:
I agree that poll fits better. I'll update!